### ISSUE TITLE
批量保存(batchSaveCommand)后,获取Entity实体id失败

### ISSUE THREAD

**Issue by hkhhh** [babyfish-ct/jimmer#17]
### 错误复现代码
```kotlin
val medias = mediaDtos.map { media ->
  new(Media::class).by {
    MediaType.valueOf(media.type)?.let {
      this.type = it
    }
    media.url?.let {
      this.url = it
    }
    this.order = media.order ?: 0
  }
}

val result = sqlClient
  .entities
  .batchSave(medias) {
    setMode(SaveMode.INSERT_ONLY)
  }

result.simpleResults.map {
  it.modifiedEntity.id
}
```
### 异常信息
<img width="1475" alt="image" src="https://user-images.githubusercontent.com/49685894/205505606-ba94f8c1-7484-4228-ade3-63ab075b05ad.png">

### 附: Media.kt
```kotlin
import com.wonderkids.api.core.enums.MediaType
import org.babyfish.jimmer.sql.*

@Entity
@Table(name = "WK_MEDIA")
interface Media {

  @Id
  @GeneratedValue(strategy = GenerationType.IDENTITY)
  val id: Long

  val type: MediaType

  val url: String

  @Column(name = "`ORDER`")
  val order: Int
}
```

**Comment by babyfish-ct** [comment]
This problem is fixed in 0.5.12, please close it after your validation

**Comment by hkhhh** [comment]
> This problem is fixed in 0.5.12, please close it after your validation

Thanks so much!
🎉🎉
